Gems/Atom/RPI/Code/Include/Atom/RPI.Public/RenderPipeline.h

```diff
@@ -98,6 +98,9 @@ namespace AZ
                                                                    const ViewType viewType = ViewType::Default);
             static RenderPipelinePtr CreateRenderPipelineForWindow(Data::Asset<AnyAsset> pipelineAsset, const WindowContext& windowContext);
 
+            // Create a render pipeline which renders to the specified attachment image
```

Comment:
Sure. I will update the comment. 

Basically for the current root pass templates, such MainPipeline.pass or the new CameraPipeline.pass, we can't just create a pass from it and make it the root pass of the render pipeline. This is because they don't have output attachment specified. 
The CreateRenderPipelineForWindow function create a SwapchainPass as root pass which provide input for the MainPipeline, in other case such as preview renderer, it creates RenderToTexture pass which provides a transient attachment as the output for the previewRenderer pass template. Other than providing output attachment, they were also responding to setup viewport and scissor.